### ISSUE TITLE
Planning ticket using pending reasons

### ISSUE THREAD

**Issue by CartCaved** [glpi-project/glpi#15035]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.7

### Bug description

Sometimes when working with a ticket, I need to pause the processing of the ticket. It may be a new ticket that should be recorded, maybe assigned, but can't or shouldn't have any work done at that time. Or a ticket that need to wait for another external activity or date (delivery, external approval, return from leave/work trip etc).

Similar use cases have been raised in for example https://glpi.userecho.com/communities/1/topics/100-plan-ticket-not-only-ticket-tasks and https://glpi.userecho.com/communities/1/topics/208-ticket-creation-delayed

I would like to use pending reasons for this purpose, but instead of the ticket being resolved after the time limit, the status is set back to processing (assigned). I would also like to be able to set a specific date instead of just a recurring schedule and have the assigned technician receive a notification when the ticket is reactivated. The status while paused could be either pending or processing planned.

This could perhaps be an extension to the rework of pending reasons tracked in #14698.

In short - the request is for
1. pending reasons to allow a date instead of just a automatic follow up schedule,
2. allow the pending schedule/date to result in status -> processing (assigned) in addition to resolved.
3. a follow up/notification when the ticket status is reverted back to processing (assigned)

Perhaps I'm misunderstanding or there is already another way to achieve a similar result but I haven't been able to find it in my searches of the issues, forum or user echo.
